joomla/joomla-cms
# [4.2] Menu item custom field

### Summary of Changes
adds a custom field to show a list of all menu items that can be selected and then displayed as a link in the frontend


### Testing Instructions
Install a prebuilt package
or
Install this PR, discover the extension and publish the plugin

Create a menuitem custom field. Check that it shows all menuitems from all menus in the select ;ist and then check on the front end that the link is displayed.


### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/177824622-d0720ecd-3bc6-424b-aed3-59fa489aa793.png)

![image](https://user-images.githubusercontent.com/1296369/177824738-7e5d63af-52e5-4c1c-b15f-54019f7315c0.png)

![image](https://user-images.githubusercontent.com/1296369/177824826-246e4cb9-95e7-4a81-ac71-046235d62de2.png)



### Documentation Changes Required
New plugin
